### ISSUE TITLE
Node3D: `Transform` inspector CTRL and SHIFT has no effect to the slider

### ISSUE THREAD

**Issue by MikeSchulze** [godotengine/godot#94979]
### Tested versions

v4.3.rc1.mono.official [e343dbbcc]

### System information

Windows 11

### Issue description

I encountered this problem because I would use the 'Page' argument for 'HSlider' but it has no effect.
Then I checked on the Godot `Node3D` inspector `Transform` to use the CTRL or SHIFT as the tooltips says.
![image](https://github.com/user-attachments/assets/6d8cf44e-73f3-434d-a796-6dd6aa190046)

But pressing CTRL or SHIFT does not work inside the inspector.
When I use the gizmo in 3D view, it rotates 15° when I press CTRL and 5° when I press SHIFT.

### Steps to reproduce

- Create a 3D Scene and add a Node3D with a mesh.
- switch to the inspector and try to use CRTL or SHIFT during using the slider (no effect)
- verify it by using the gizmo inside the 3d view (it works)

### Minimal reproduction project (MRP)

n/a

**Comment by Giganzo** [comment]
The problem seems to be when grabbing the slider handle/grabber. 
If you click in the the box and drag then ctrl, or shift works as described in the tooltip. Is it same for you?

**Comment by MikeSchulze** [comment]
You are right, if I'm clicking in the box the CTRL and SHIFT is working, just when using the scrollbar it has no effect.
But it handles it different as the Gizmo control, so there is an inconsistency.
The Gizmo + CTRL rotates by 15 degree and in the box by 1 degree.
The HSlider works also in this way, clicking in the box + CTRL or SHIFT works fine, just when the mouse is over the scrollbar it has no effect. 

The tooltip do only appear over the box and not above the slider, but it is confusing and inconsistent.  :)
